src/wmtk/TetMesh.cpp

```diff
@@ -175,9 +179,11 @@ Tuple TetMesh::face_tuple_from_id(long id) const
 
     if (lvid < 0 || leid < 0 || lfid < 0) throw std::runtime_error("face_tuple_from_id failed");
 
-    Tuple f_tuple = Tuple(lvid, leid, lfid, t, get_cell_hash_slow(t));
+    ConstAccessor<long> hash_accessor = get_const_cell_hash_accessor();
```

Comment:
in the long term this construction / passing of accessors could become more prevalent (even `face_tuple_from_id` might want to haev a hash accessor

Reply:
yes, I guess we might end up having that everywhere